openapi.yaml

```diff
@@ -5593,6 +5589,20 @@ components:
       description: >
         A [shipping carrier](https://www.shipengine.com/docs/carriers/setup/),
         such as `fedex`, `dhl_express`, `stamps_com`, etc.
+    lat:
+      format: double
+      type: number
+      example: 48.842608
+      description: >-
+        The latitude of the point. Represented as signed degrees. Required
+        if long is provided. http://www.geomidpoint.com/latlon.html
+    long:
```

Comment:
Why are not using here `$ref: '#/components/schemas/long'`?

Reply:
this one is 'components/schemas/long' and where the other descriptions came from